### ISSUE TITLE
LEAP: Is `zone: us-central1-a` correct, or should it be regional as `zone: us-central1`?

### ISSUE THREAD

**Issue by consideRatio** [2i2c-org/infrastructure#1258]
@yuvipanda did you migrate some clusters from zonal to regional recently?

I tried to use `python deployer use-cluster-credentials leap` recently but got an error that went away by updating the leap/cluster.yaml's zone to be a region instead of a zone.

https://github.com/2i2c-org/infrastructure/blob/45e51afe8335e9f8efc700e3a95f3c21dd2bf36a/config/clusters/leap/cluster.yaml#L7

Is there a change to be made here, and also, in other clusters about this?

**Comment by GeorgianaElena** [comment]
I ran into this too. I believe it's https://github.com/2i2c-org/infrastructure/pull/1251 that changed the clusters and we should definitely merge that asap.

I just changed locally `us-central1-c` to `us-central1` in https://github.com/2i2c-org/infrastructure/blob/45e51afe8335e9f8efc700e3a95f3c21dd2bf36a/config/clusters/leap/cluster.yaml#L7 to be able to run `python deployer use-cluster-credentials leap` and access the logs successfully.
